bindgen/codegen/mod.rs

```diff
@@ -4236,6 +4251,9 @@ impl CodeGenerator for Function {
         if should_wrap {
             let name = canonical_name.clone() + ctx.wrap_static_fns_suffix();
             attributes.push(attributes::link_name::<true>(&name));
+        } else if is_function_macro && !has_link_name_attr {
+            let name = canonical_name.clone() + "__macro";
```

Comment:
I think this `__macro` suffix should be configurable, maybe reusing the suffix for static wrappers is a good idea, you could rename it to `wrapper_suffix` or something.